ddl/column.go

```diff
@@ -316,13 +378,13 @@ func doModifyColumn(t *meta.Meta, job *model.Job, newCol *model.ColumnInfo, oldN
 	if pos.Tp == ast.ColumnPositionAfter {
 		if oldName.L == pos.RelativeColumn.Name.L {
 			// `alter table tableName modify column b int after b` will return ver,ErrColumnNotExists.
-			job.State = model.JobStateCancelled
+			job.State = model.JobStateRollingback
```

Comment:
Why change it to JobStateRollingback？

Reply:
If an error occurs here, it will not be a rollback.